tests/flask/test_flask_route.py

```diff
@@ -0,0 +1,16 @@
+import pytest
```

Comment:
If you're gonna put the example at `flask_api`, then this dir should be `tests/flask_api`